Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,54 @@
-﻿namespace Logger;
-
-public static class BaseLoggerMixins
-{
-
-}
+﻿using System.Globalization;
+
+namespace Logger;
+
+public static class BaseLoggerMixins
+{
+    public static void Error(this BaseLogger? logger, string message, params object[] args)
+    {
+        if (logger == null)
+        {
+            throw new System.ArgumentNullException(nameof(logger));
+        }
+        else
+        {
+            logger.Log(LogLevel.Error, string.Format(CultureInfo.CurrentCulture, message, args));

```

Comment:
I think the following could be different
```suggestion
            logger.Log(LogLevel.Error, string.Format(CultureInfo.CurrentCulture, message, args));
```
to
```suggestion
            string formatMess = string.Format(message, args);
            logger.Log(LogLevel.Error, formatMess);

```